word_guess.rb

```diff
@@ -0,0 +1,101 @@
+#https://github.com/Ada-C7/Word-Guess
+require 'faker'
+
+class WordGame
+
+  def initialize
+    @lives = 9
+    @word = Faker::Cat.unique.name.downcase
+    @underscore_array = Array.new( @word.length, "_" )
+    @word_array = @word.split("")
+
+  end
+
+  def convert_string_to_array
+    @word_array = @word.split("")
+    # puts "#{@underscore_array}"
+    # puts "#{@underscore_array.join(" ")}"
+    @underscore_string = @underscore_array.join(" ")
+    # puts "#{@word_array}"
+    # puts "#{@word_array.join("")}"
+    @word_string = @word_array.join("")
+  end
+
+def big_loop
```

Comment:
Watch your indentation - this method should be at the same indentation level as the others in this class. Clean indentation makes code much easier to read, which can help keep you from getting lost in your loops.